improve777/RotateImage.kt

```diff
@@ -0,0 +1,37 @@
+class RotateImage {
+    fun rotate(matrix: Array<IntArray>): Unit {
+        val n = matrix.size
+
+        for (i in 0 until n) {
+            for (j in 0 until i) {
+                matrix[i][j] = matrix[j][i].also { matrix[j][i] = matrix[i][j] }
```

Comment:
저도 이거 대연님이 말씀하신거 생각나서 해보려고 했는데, 감이 안왔었네요 ㅋㅋ
```suggestion
                matrix[i][j] = matrix[j][i].also { it = matrix[i][j] }
```
이렇게도 되겠군요